### PR TITLE
Add Hacks RFC to Standards

### PR DESCRIPTION
**Pull request by paololim**

### CODE REVIEW

File: Hacks.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+#### Issues with this process:
+
+- Difficult to actually find the hacks in the code
+- Difficult to verify that a hack was actually removed
+- Hacks get “paid down” sometimes (e.g. “December Projects”), but difficult to track and make visible that these actually happened
+
+## Proposal
+
+#### Create a single package in each application that ends with “.hacks”
```
[the remark this answers is not in the document]
Comment (paololim) [reply Hacks.md]: Good callout here - I think a single `Hacks.scala` file containing both an object and injectible singleton is good.

**Comment by paololim** [comment]
@mbryzek Updated.

File: Hacks.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+- Difficult to actually find the hacks in the code
+- Difficult to verify that a hack was actually removed
+- Hacks get “paid down” sometimes (e.g. “December Projects”), but difficult to track and make visible that these actually happened
+
+## Proposal
+
+#### Create a single `Hacks.scala` in each application
+
+- Contains at least one of: `object Hacks` or injectible `case class Hacks`
```
Comment (kubukoz) [inline Hacks.md]: Why does it have to be a case class?

Comment (paololim) [reply Hacks.md]: doesnt have to be. updating

**Pull request merged**